### PR TITLE
remove desired capabilities

### PR DESCRIPTION
**Pull request by alexgillcma**
* desired_capabilities deprecated in latest selenium version
* replaced by "set_capabilities" method on options

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,13 +4,15 @@ Changelog
 The format is based on `Keep a Changelog <https://keepachangelog.com/en/1.0.0/>`_,
 and this project adheres to `Semantic Versioning <https://semver.org/spec/v2.0.0.html>`_.
 
-`Unreleased <https://github.com/cmagovuk/selene-core/compare/v1.0.0...master>`_
+`Unreleased <https://github.com/cmagovuk/selene-core/compare/v1.0.1...master>`_
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- 
+Fixed
```
Comment (dayalstrub-cma) [inline CHANGELOG.rst:10]: should we release this as 1.0.2 if it's a necessary fix for selenium updates?

Comment (alexgillcma) [reply CHANGELOG.rst:10]: yes, I think so

**Review by scostantinicma — APPROVED** [review]
LGTM

**Comment by alexgillcma** [comment]
@dayalstrub-cma are you okay to approve this? Then I will release and tag

**Pull request merged**